### ISSUE TITLE
Browser throws errors 

### ISSUE THREAD

**Issue by johnryan** [gianu/react-fittext#9]
Seeing errors when i load the page in my browser
```
Warning: getDefaultProps is only used on classic React.createClass definitions. Use a static property named `defaultProps` instead.
warning.js:36 Warning: Something is calling a React component directly. Use a factory or JSX instead. See: http://fb.me/react-legacyfactory
warning.js:48 Warning: ReactFitText(...): No `render` method found on the returned component instance: you may have forgotten to define `render` in your component or you may have accidentally tried to render an element whose type is a function that isn't a React component.
warning.js:36 Warning: Don't set the props property of the component. Mutate the existing props object instead.
ReactCompositeComponent.js:153 Uncaught TypeError: Can't add property context, object is not extensible
```

**Comment by gianu** [comment]
Thanks for reporting this @johnryan . I'll take a look at it today.

**Comment by robgietema** [comment]
I got the same error, I'm using ES6.

**Comment by sevenfoxes** [comment]
I get this issue as well.

**Comment by jonlong** [comment]
Hey sweet — looks like this got fixed in #8!

@gianu I think this error is coming from the version on NPM, which looks like it hasn't been updated yet.

**Comment by gianu** [comment]
@jonlong I just published a new version with the change on #8 . Let me know if this fix the issue.

**Comment by jonlong** [comment]
@gianu Works! Thanks for the quick fix!
